src/main/xar-resources/modules/view.xql

```diff
@@ -1,24 +1,33 @@
 xquery version "3.1";
 
-import module namespace app="http://exist-db.org/xquery/app" at "app.xql";
-import module namespace config="http://exist-db.org/xquery/apps/config" at "config.xqm";
 
 import module namespace templates="http://exist-db.org/xquery/html-templating";
 import module namespace lib="http://exist-db.org/xquery/html-templating/lib";
 
-declare option exist:serialize "method=html5 media-type=text/html";
+(: The following modules provide functions which will be called by the templating :)
+import module namespace config="http://exist-db.org/xquery/apps/config" at "config.xqm";
+import module namespace app="http://exist-db.org/xquery/app" at "app.xql";
+
 
-let $config := map {
+declare namespace output="http://www.w3.org/2010/xslt-xquery-serialization";
+
+declare option output:method "html5";
```

Comment:
Minor issue, but there's no `html5` method in the serialization spec. eXist lets this through for backwards compatibility. The spec-compliant declaration is:

```
declare option output:method "html";
declare option output:html-version "5.0";
```

Reply:
good catch! This needs to be changed in documentation app as well.